__tests__/endpoints.test.js

```diff
@@ -74,6 +74,22 @@ describe('NC news endpoint tests',()=>{
                     expect(body.msg).toBe("article does not exist")
                 })
             })
+            test('Respond with 200 with the correct article and its comment count',()=>{
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(({body:{article}})=>{
```

Comment:
Lovely use of nested destructuring here. 
If you wanted to further stretch your jest knowledge you could try looking at the `.toMatchObject` matcher, which would be a nice refactor for this test.